theano/gpuarray/linalg.py

```diff
@@ -5,15 +5,21 @@
 import warnings
 
 from theano import Op
+
 from theano.gpuarray import basic_ops, GpuArrayType
 
+from pygpu.gpuarray import GpuKernel, GpuArray
+from string import Template
+
```

Comment:
Remove these 3 lines as GpuKernel, GpuArray and Template are not used anymore in this file (it currently makes flake8 fail).